<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>931</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Решить задачу</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>71</x>
        <y>21</y>
        <width>260</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Набор чисел из файла</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditFirst">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Ввести набор чисел из файла</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>260</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Результат</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditFirst_2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>Записать результат в файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>261</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Результат</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditSecond_2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSecongInfile">
         <property name="text">
          <string>Записать результат в файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_24">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Очередь вначале</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdiSecond">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonInputQueue">
         <property name="text">
          <string>Ввести очередь из файла</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter_4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>562</width>
        <height>143</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Покупатели, которых обслужили</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditM"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Касса закроется через (в секундах)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditT"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Покупатели, которые встали в очередь</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditK"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_23">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Время на одного покупателя(в секундах)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditMC"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButtonSecondSolve">
       <property name="text">
        <string>Решить задачу</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>261</width>
        <height>152</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_25">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_26">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Стек</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditThird">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonThirdInput">
         <property name="text">
          <string>Ввести стек из файла</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButtonThirdSolve">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Решить задачу</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>60</y>
        <width>259</width>
        <height>114</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_28">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Результат</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditThird_2">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>261</width>
        <height>152</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_32">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_33">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Стек</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditFourth">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFourthInput">
         <property name="text">
          <string>Ввести стек из файла</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButtonFourthSolve">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>130</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Решить задачу</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>70</y>
        <width>260</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_34">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_31">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Результат</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditFourth_2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFourth">
         <property name="text">
          <string>Записать результат в файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>881</width>
      <height>242</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Выбор задачи&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Задан набор целых чисел.После каждого нечетного вставить четный на 1 больше предыдущего</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Выбрать</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Вначале в очереди было n покупателей.
Кассир обслужил m покупателей, после чего еще k покупателей стали в очередь, и кассир объявил,
что через t мин касса закрывается на обед. Успеет ли кассир обслужить всех покупателей</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Выбрать</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string> Подсчитать количество элементов стека, у которых равные &quot;соседи&quot;.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Выбрать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Просматривая  элементы  стека по тройкам, поменять крайние местами</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Выбрать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>550</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Бирило Роман 5 группа,1 курс</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
